app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_out = command.split()[1]
+    file_in = command.split()[2]
+    with (open(file_out) as file_out,
+          open(file_in, "w") as file_in):
+        file_in.write(file_out.read())
```

Comment:
The logic here is incorrect. You should read from the source file and write to the destination file. Swap the roles of `file_out` and `file_in` in the `write` method call.